docs/dev/index.rst

```diff
@@ -25,15 +25,36 @@ In the following of this document you will find information related to the speci
 of making a development workflow for `sisl`.
 
 
-
+.. _devSummary:
 Summary of development process
 ------------------------------
 
+You are free to choose your development environment, but we recommend using a conda virtual
+environment because things are very easy to set up. To create one, install `miniforge`_
+and then create a new python environment. E.g. to create an environment named ``sisl_dev``
+with the latest python version do:
+
+.. code:: bash
+
+   conda create -n sisl_dev python
```

Comment:
Why not have everything installed in this line? At least for the conda based solution?